Fixup mksrcinfo

mikkeloscar opened the pull request:
This adds a couple of missing things to `mksrcinfo`. `-h` was not implemented and the `error` function used in `die` was missing.

I have also defaulted the output to `stdout` rather than `$PWD/.SRCINFO`. I think this is a better default (and useful to me :)).

falconindy commented:
Hi, I'm interested in the first 3 patches, but not the last. If you want to make your argument for it being a better default, then do so, and explain your use case. I picked $PWD/.SRCINFO as the default because the number 1 use case for mksrcinfo will be generating .SRCINFO files for soon-to-be git repositories in the AUR. The workflow will essentially be:

1) Modify PKGBUILD, et al.
2) mksrcinfo
3) commit and push

If you want to write the .SRCINFO to stdout, then you can still do that in the current world, by using -o /dev/stdout

mikkeloscar commented:
Hi, I have fixed the error in the optstring and removed the stdout stuff.

It make sense to have it default to $PWD/.SRCINFO given the intended workflow.

falconindy commented:
Thanks!
